README.reproducible.md

```diff
@@ -0,0 +1,148 @@
+#Armory Reproducible Build Documentation
+
+##Gitian
+
+Gitian is the build system that Bitcoin Core uses for reproducible builds.
+Armory has adopted it for OS X and Windows reproducible builds. This guide
+assumes you are using KVM as the virtualization technology to make the VMs
+that Gitian uses to build in, but gitian-tools also supports LXC.
+
+The following steps will allow you to build Armory reproducibly using Gitian.
+
+1. Obtain gitian-tools from [Devrandom's GitHub][Devrandom's GitHub].
```

Comment:
Are the links working? Maybe it's just Github but I see "Devrandom's Github" twice. Same idea goes for all the other links.